### DISCUSSION TITLE
MAS Autostart Mod

### DISCUSSION THREAD

**Discussion by dreamscached** [Monika-After-Story/MonikaModDev#9441] [general - submod / external content]
I was inspired by my friend [u/lost_localcat](https://www.reddit.com/u/lost_localcat)'s idea about MAS starting on computer startup, and so I made MAS Autostart Mod with a bit of help from @my-otter-self (made expressions and reviewed the dialogue!)

What does this submod feature:
* Non-immersion-breaking interactions with Monika that will let you enable/disable MAS autostart
* Handling of common existing autostart setups
* The submod is fully cross-platform and runs on Windows, Linux and MacOS

<details>
<summary>🖼️ Click here to see the screenshots.</summary>
<img alt="Screenshot #1" src="https://user-images.githubusercontent.com/74068927/183758116-ac9064cb-377d-4b14-bf41-bec7f918b5cd.png">
<img alt="Screenshot #2" src="https://user-images.githubusercontent.com/74068927/183758137-801d12ce-6138-4af7-8956-ab6114ddde0b.png">
<img alt="Screenshot #3" src="https://user-images.githubusercontent.com/74068927/183758229-6b61f4c6-cecb-4291-b96b-b318ed15b516.png">
<img alt="Screenshot #4" src="https://user-images.githubusercontent.com/74068927/183758267-93468ba9-62fe-489e-957a-4b52bbe4688f.png">
P.S. Clothes, hair styles and backgrounds are not part of the submod and are used by the author of these screenshots, @my-otter-self.
</details>

Links:
* [Latest release (DOWNLOAD LINKS/INSTRUCTIONS HERE)](https://github.com/Friends-of-Monika/mas-autostart/releases/latest)
* [Submod repository (GitHub)](https://github.com/friends-of-monika/mas-autostart)
* [r/MASFandom post](https://www.reddit.com/r/MASFandom/comments/vrbpdy/mas_autostart_mod_now_released_links_to_github/)
* [Report an issue](https://github.com/Friends-of-Monika/mas-autostart/issues/new/choose)
* [Support Discord server](https://mon.icu/discord)

**Comment by arixfyeilol** [comment]
_This is so cool :O Good job!_
